backend/api/serializers.py

```
@@ -158,4 +158,22 @@ def get_points_value(self, obj):
+    class Meta:
+        model = WebcamRecording
+        fields = [
+            'id',
+            'video',
+            'recorder',
+            'filename',
+            'recording_date',
+            'upload_status',
+            'upload_completed_at',
+            'recording_url'
+        ]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)